library/src/main/java/com/novoda/simplechromecustomtabs/SimpleChromeCustomTabs.java

```diff
@@ -67,7 +59,6 @@ public WebNavigator withFallback(NavigationFallback navigationFallback) {
      * Provides a {@link IntentCustomizer} to be used to customize the Chrome Custom Tabs by attacking directly to
      * {@link SimpleChromeCustomTabsIntentBuilder}
      *
-     * @param intentCustomizer
```

Comment:
https://github.com/novoda/simple-chrome-custom-tabs/pull/37#discussion_r107660633 👎 

This is javadoc it is used to generate helper documentation, if you are using javadoc and you have the paramater for the method then you should have the javadoc for that.

If you want to change this, then add a description to the param , do not remote it (making it worse)